src/cft.ts

```diff
@@ -4,22 +4,25 @@ import { selectTestEnvironment } from "./environments/selectTestEnvironments";
 import { getSpyFactory } from "./spies/selectSpyFactory";
 import { MethodCall, MethodSpy } from "./spies/spyTypes";
 import { CftRequest } from "./types";
+import { setDefaults } from "./defaults";
 
-export const cft = (request: CftRequest = {}) => {
+export const cft = (_request: Partial<CftRequest>) => {
```

Comment:
> `_request`

Just a thought: this _might_ be a little more clear as `rawRequest`, which is the naming convention I normally go by. But totally fine if you prefer `_request` 😄 not a blocker at all.

Reply:
Changed